templates/grub-pxelinux_config

```diff
@@ -107,7 +107,9 @@ if [ -d "$MOUNT_POINT_"/boot/release ] ; then
 fi
 
 # adjust ldbsd.com bootline
-sed -i -e 's#\(.*kernel .*/ldbsd.com\)#\1 set image /boot/addons/bsd4grml/bsd.rd`echo Type "boot" to load MirOS bsd4grml via keeppxe#' "$TFTPD_DATA_DIR_/"addon_*_bsd.cfg
+if ls "$TFTPD_DATA_DIR_/"addon_*_bsd.cfg >/dev/null 2>&1; then
```

Comment:
Untested yet, but LGFM. Out of curiosity, why "ls" and not "[ -e $FILE ]" (or better "[ -f $FILE ]")?

Reply:
Because we need to test for multiple files with the `*` wlldcard and `test` respectively `[` doesn't support multiple arguments

Reply:
Ah, makes sense! :man_facepalming: Not enough :coffee: in the early morning! :)
Sorry for the noise!